packages/manager/tools/webpack-config/src/webpack.common.js

```diff
@@ -23,9 +24,21 @@ const cacheLoader = {
   },
 };
 
+const readClosestPackage = (entry) => {
```

Comment:
i don't think you need to do that you just need to define your aliases in your own webpack config by extending this default configuration

Reply:
This is exactly what they - the apps that import the IAM module - do [here](https://github.com/ovh/manager/pull/8855/files#diff-7efd7f75a5e0dac4f9df27bb7504c7c1d7ac723943f160a3d211f30f3ccdef55R53) and [here](https://github.com/ovh/manager/pull/8855/files#diff-bca528868988736434df1ac5cf88ff4553ae365eac1db02cdc4bad69d41aab6fR13). Because we use a common webpack file to do common stuff, and because module could be imported by many apps (IAM and Dedicated in our case), I created a common option instead of duplicating the code

That way :
- The module defines its own aliases, which is the best thing to do since it uses the aliases
- Applications that import the module set the common `resolveAlias` option because the module requires them to (think of it as a peer dependency) and don't know about the details
- The common webpack file only handles the option, it is now pure algorithm (I was asked by @antleblanc to remove all business logic in a previous version, which I did)